docs/setup-robusta/openshift.rst

```diff
@@ -0,0 +1,68 @@
+.. _openshift-permissions:
+
+OpenShift
+========================================
+
+OpenShift is supported via the Helm chart.
```

Comment:
What does this mean? Perhaps phrase as:

Robusta supports OpenShift. To use Robusta on OpenShift, you should enable the following Helm values:

...

And I'd put the warning below after the Helm values. Lead with the main thing, then add the caveat.